ICSharpCode.Decompiler/CSharp/OutputVisitor/CSharpAmbience.cs

```diff
@@ -190,6 +192,7 @@ TypeSystemAstBuilder CreateAstBuilder()
 		{
 			TypeSystemAstBuilder astBuilder = new TypeSystemAstBuilder();
 			astBuilder.AddTypeReferenceAnnotations = true;
+			astBuilder.AddResolveResultAnnotations = true;
```

Comment:
Do we still use the old type reference annotations anywhere, or did we completely switch to resolve results?

Reply:
I just checked on the master branch. Looks like we don't use them anymore.